src/components/fx/hover.js

```diff
@@ -580,24 +580,27 @@ function createHoverText(hoverData, opts, gd) {
             ltext = label.selectAll('text').data([0]);
 
         lpath.enter().append('path')
-            .style({
-                fill: commonLabelOpts.bgcolor || Color.defaultLine,
-                stroke: commonLabelOpts.bordercolor || Color.background,
-                'stroke-width': '1px'
-            });
+            .style({'stroke-width': '1px'});
+
+        lpath.style({
```

Comment:
nicely done.

Reply:
it was so hard to figure out what flags would happen in a particular restyle call before... I *thought* I got the same ones for `layout.hoverlabel` but [this test](https://github.com/plotly/plotly.js/blob/11bb997e78128b9f47db2ed392982f82a026e589/test/jasmine/tests/hover_label_test.js#L1201) was failing until I made this change so it must have been hitting some other conditional I didn't notice.